shuffle-kubernetes-operator/src/test/java/com/alibaba/flink/shuffle/kubernetes/operator/controller/RemoteShuffleApplicationControllerTest.java

```diff
@@ -428,6 +428,7 @@ public void testShuffleManagerUpdate() throws Exception {
                 Duration.ofSeconds(30));
     }
 
+    @Ignore("Temporarily ignore.")
```

Comment:
Sorry about I'm not clear about why we ignore the test here. Is the change will break it or is the test not stable?

Reply:
The test is unstable, this change is in a separated hotfix commit.